Lib/distutils/command/check.py

```diff
@@ -135,6 +135,7 @@ def _check_rst_data(self, data):
                           error_handler=settings.error_encoding_error_handler)
 
         document = nodes.document(settings, reporter, source=source_path)
+        # the include and csv_table directives need this to be a path
```

Comment:
This comment looks like it would be more helpful next to the assignation on line 123?

Reply:
done